src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetDiskFreeSpace.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [LibraryImport(Libraries.Kernel32, EntryPoint = "GetDiskFreeSpaceW",  SetLastError = true, StringMarshalling = StringMarshalling.Utf16)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static partial bool GetDiskFreeSpace(string drive, out int sectorsPerCluster, out int bytesPerSector, out int numberOfFreeClusters, out int totalNumberOfClusters);
```

Comment:
DWORD maps to uint, I think: https://learn.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#common-windows-data-types

Reply:
should it not be `out uint` then?

Reply:
@jkoritzinsky do we prefer to map LPDWORD to `out uint` or `uint*` (as I see below)

I don't see it mentioned in https://learn.microsoft.com/en-us/dotnet/standard/native-interop/best-practices which I'm trying to update as I learn.

Reply:
My opinion is: `out uint` wouldn't always be applicable, note that it's marked as `[out]` in this case. You would potentially want `ref uint` sometimes. I think `uint*` would always be an okay option, since that's what it actually is. `out/ref uint` would just compile to `uint*` in the generated method for `[LibraryImport]`.
I've changed it to `uint*`, since I think that fits best here.